File: TP/TP1/Code/4F.cpp
/* Programmer une calculatrice proposant les opérations classiques */
#include <iostream>

int main()
{
  int a;
  int b;
  char c;
  
  std::cout << "Saisi une valeur : " << std::flush;
  std::cin >> a;
  std::cout << "Saisi une deuxième valeur : " << std::flush;
  std::cin >> b;
  std::cout << "Choisi un operateur parmi : '+' , '-' , '*' et '/' " << std::flush;
  std::cin >> c;
  
  switch (c) {
  case '+':
    std::cout << a << " + " << b << " = " << a + b << std::endl;
    break;
  case '-':
    std::cout << a << " - " << b << " = " << a - b << std::endl;
    break;
  case '*':
    std::cout << a << " * " << b << " = " << a * b << std::endl;
    break;
  case '/':
    std::cout << a << " / " << b << " = " << a / b << std::endl;
    break;
  default:
    std::cout << "Unknown error !" << std::endl;
  }
  
  return 0;
}

File: TP/TP1/Code/Python/4F.py
#!/usr/bin/python
# Programmer une calculatrice proposant les opérations classiques
print('Donne moi une valeur : ', end='')
foo = int(input())
print('Donne moi une autre valeur : ', end='')
bar = int(input())
print('Donne moi l\'opérateur ( + | - | * | / ) : ', end='')
baz = str(input())

if baz == '+':
    print(foo, '+', bar, '=', foo + bar)
elif baz == '-':
    print(foo, '-', bar, '=', foo - bar)
elif baz == '*':
    print(foo, '*', bar, '=', foo * bar)
elif baz == '/':
    if bar == 0:
        print('La division par zéro')
    else:
        print(foo, '/', bar, '=', foo / bar)
else:
    print('Une erreur inconnue')
